src/Http/Http.Extensions/src/EndpointParameterMetadataContext.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Http.Metadata;
+
+/// <summary>
+/// Represents the information accessible during endpoint creation by types that implement <see cref="IEndpointParameterMetadataProvider"/>.
+/// </summary>
+public class EndpointParameterMetadataContext
```

Comment:
Happy to if that's the standard.

Reply:
I'm assuming it's because we haven't been very strict about adding sealed to classes in the past but are trying to be more vigilant these days. Is there any reason someone wants to provide a custom implementation of this type? If not we can always remove sealed later which is non-breaking, but adding sealed after the fact is breaking.